content/doc/developer/plugin-development/build-process.adoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: developersection
+title: Plugin Build Process
+references:
+- url: ../updating-parent/
+  title: Guide to Updating Your Maven Parent POM
+- url: https://github.com/jenkinsci/plugin-pom/
+  title: Plugin parent POM 2.x and up
+- url: https://github.com/jenkinsci/maven-hpi-plugin/
+  title: Maven HPI Plugin
+---
+
+Most Jenkins plugins are built with link:https://maven.apache.org[Maven]. The link:https://github.com/jenkinsci/maven-hpi-plugin/[Maven HPI plugin] does the heavy lifting, such as bundling plugins in the HPI/JPI archive format, or running a debug Jenkins instance with the plugin.
+
+Jenkins plugins typically use the Jenkins plugin parent POM, which provides a sensible default configuration for the build.
+Up to Jenkins 1.645, the plugin POM was kept in sync with Jenkins releases, so that the minimum required Jenkins version for a plugin determined the versions of the tools used to build the plugin.
+As of Jenkins 1.646, they've been decoupled:
+Plugin POM 2.0 and newer allow specifying the core version dependency independent of its version, so that even plugins compatible with older Jenkins releases can benefit from fixes and improvements in the parent POM.
```

Comment:
The uncoupling happened two years ago — it's probably not worth mentioning the historic info (IMO).

Reply:
Still relevant in plenty of plugins with old baselines.

Reply:
I would explicitly say that Plugin POM 2.x is recommend and required for new plugin hosting requests, WDYT? CC @slide

Reply:
This is literally the next paragraph.

Reply:
> Still relevant in plenty of plugins with old baselines.

Then I think I'm not understanding who this page is aimed at.   If it's new plugin developers, they'll get 2.x (or preferably 3.x) from the archetypes.  If it's existing plugin developers, I'm not sure why they'd read this page but most likely it would be because someone's linked them to the next section, telling them about the benefits of updating to a newer parent version.

My point was that the historic information is perhaps interesting to Jenkins core developers, but to everyone else it's irrelevant, and a waste of time for people to read.

Reply:
It is reference documentation. It describes how things are.

Reply:
Notably, if you follow hosting requests, you will see new plugins still get created with years old parent POMs. (1.x or the first few releases of 2.x.) It's far from irrelevant today.